Source/WebKit/GPUProcess/graphics/RemoteGraphicsContextGLFunctionsGenerated.h

```diff
@@ -1482,11 +1482,11 @@
         assertIsCurrent(workQueue());
         m_context->setDrawingBufferColorSpace(arg0);
     }
-    void paintRenderingResultsToPixelBuffer(CompletionHandler<void(RefPtr<WebCore::PixelBuffer>&&)>&& completionHandler)
+    void paintRenderingResultsToPixelBuffer(bool arg0, CompletionHandler<void(RefPtr<WebCore::PixelBuffer>&&)>&& completionHandler)
```

Comment:
`arg0` means `flipY` right? Should it be named as such?

Reply:
> `arg0` means `flipY` right? Should it be named as such?

Also generated by `generate-gpup-webgl`.